src/test/java/com/wikia/webdriver/elements/oasis/components/templateclassificiation/TemplateClassification.java

```diff
@@ -103,4 +95,16 @@ public TemplateClassification selectQuoteTemplate() {
 
     return this;
   }
+
+  public TemplateClassification compareTemplateTypes() {
```

Comment:
Please move the assertion out of this method, and put it in the test.

You can return `String` as a template type, and make assertion directly in the test - it will be more transparent, what is the expected bevaiour